src/main/java/org/wildfly/security/auth/spi/AuthenticatedRealmIdentity.java

```diff
@@ -35,6 +36,13 @@
     Principal getPrincipal();
 
     /**
+     * Returns the roles for this identity.
+     *
+     * @return A string set containing the roles for this identity or an empty set if this identity has no roles.
```

Comment:
Should be no leading caps and no trailing punctuation on ``@return``.  See https://dmlloyd.github.io/presentations/coding-standards/index.html#/8 for more info.